### ISSUE TITLE
Update URLs!

### ISSUE THREAD

**Issue by OriolAbril** [arviz-devs/arviz#2012]
The python documentation is now hosted on readthedocs (with automatic multiple version suport) and at the custom https://python.arviz.org domain. There are several things to update:

* [x] Delete the azure doc building job
* [x] Update the intersphinx mappings of projects using it
  * [x] pymc-devs/pymc
  * [x] pymc-devs/pymc-examples 
* [x] Delete the content in the `gh-pages` branch. If possible make it an automatic redirect to python.arviz.org ([this](https://github.com/OriolAbril/oriolabril.github.io/blob/main/index.html) could work), otherwise make it a single page website with a link to python.arviz.org
* [x] Same for the https://github.com/arviz-devs/arviz-devs.github.io repo

**Comment by OriolAbril** [comment]
I have added a warning banner for now on the github docs (that isn't shown in the readthedocs ones) with a link to the readthedocs ones (at python.arviz.org)

**Comment by charlesdong1991** [comment]
I'd like to work on it if the issue is still yet taken by other folks!

**Comment by OriolAbril** [comment]
It is still pending. Thanks for offering to help. 

We should start with updating the intersphinx mappings and other urls of related projects. I am not completely sure what is the reach of all this. We have already updated our github, twitter and opencollective, but there are probably links in the numfocus website that point to the old python docs, or in the matplotlib/bokeh/xarray ecosystem pages... We shouldn't kill the github pages version until we have warned and hopefully updated all these. pymc and pymc-examples docs also have the old link in the intersphinx mapping, maybe others. If they refer to ArviZ "project wide" they should be updated to `www.arviz.org` if they are about the python version only they should point to `python.arviz.org` (no `/en/latest/`)

The 4th point can happen in parallel to this, it is a single page website, so converting it to a redirect should give no problems.

The other two need significant advance in the update of external references to our website before we can actually do it.
